lists_and_tuples/task_02.py

```diff
@@ -0,0 +1,10 @@
+import random 
+import string
+
+all_characters = list(string.ascii_lowercase + string.ascii_uppercase + string.digits)
+
+password = ""
+for _ in range(8):
+    password += random.choice(all_characters)
```

Comment:
Possible improvement
Check on  choices `k` parameter

```suggestion
random_pass = random.choices(all_letters, k=8)
password = ''.join(random_pass)
```